index.html

```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
+              <a href="#">
+                <svg width="auto" height="auto" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" fill="#505F79"/>
+                </svg>
+              </a>
+            </li>
+
+          </ul>
+
+        </div>
+
+        <div id="about-myself-top-block-apart-action">
+          <button id="block-apart-action-button" type="button">Get my resume</button>
+        </div>
+
+      </div>
+
+
+      <div id="about-myself-skill-list">
+
+        <div id="about-myself-skill-list-frame-12">
+
+          <div id="about-myself-skill-list-frame-7">
+
+            <button id="about-myself-skill-list-button" type="button">
+              <span id="about-myself-skill-list-union">
+                <svg width="auto" height="auto" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                </svg>
+              </span>
+            </button>
+
+            <a id="about-myself-skill-list-skills">Languages</a>
+          </div>
+
+          <div id="about-myself-skill-list-language-list">
+
+            <div id="about-myself-skill-list-frame-92" class="about-myself-skill-list">
+
+              <div id="about-myself-skill-list-icon-1">
+              </div>
+
+              <a id="about-myself-skill-list-text-1" class="about-myself-skill-list-text">JavaScript</a>
+
+            </div>
+
+            <div id="about-myself-skill-list-frame-10" class="about-myself-skill-list">
+
+              <div id="about-myself-skill-list-icon-2">
+              </div>
+
+              <a id="about-myself-skill-list-text-2" class="about-myself-skill-list-text">HTML</a>
+
+            </div>
+
+            <div id="about-myself-skill-list-frame-11" class="about-myself-skill-list">
+
+              <div id="about-myself-skill-list-icon-3">
+              </div>
+
+              <a id="about-myself-skill-list-text-3" class="about-myself-skill-list-text">CSS</a>
+
+            </div>
+
+          </div>
+
+        </div>
```

Comment:
- [ ] Kindly make sure the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`